File: vm_Assignments/vm/Beginning/L2.6/Pythagorean.py
def Pythagorean():
    for a in range(1, 100):
        for b in range(1, 100):
            for c in range(100):
                if(a < b) and (a * a) + (b * b) == (c*c):
                    print("a =", a, "b =", b, "c =", c)
Pythagorean()

File: vm_Assignments/vmc/L2.6/Pythagorean.c
#include<stdio.h>
void Pythagorean()
{
    for(int a=1; a<100; a++)
    {
        for(int b=1; b<100; b++)
        {
            for(int c=1; c<100; c++)
            {
                if(a<b && (a*a)+(b*b)==(c*c))
                printf("a=%d  b=%d  c=%d\n", a, b, c);
            }
        }
    }
}
int main()
{
    Pythagorean();
}
